secure_message/repository/modifier.py

```diff
@@ -76,11 +77,23 @@ def add_unread(message, user):
         return res
 
     @staticmethod
-    def del_unread(message, user):
+    def del_unread(message, user, session=db.session):
```

Comment:
Is there a reason to make session a parameter if it's never passed in?
Also, it might be worth renaming the method now.

Reply:
Renamed the method to something more descriptive and accurate.  
I tried moving session into the function, using db.session directly and other stuff but pylint complains that with a bunch of `Instance of 'scoped_session' has no 'add' member`.  Not sure what to do to fix that, any thoughts?

Reply:
Survey runner disables the

Reply:
Another option could be to add `scoped_session` to the `ignored-classes` property in .pylintrc (see https://stackoverflow.com/questions/28193025/pylint-cant-find-sqlalchemy-query-member)